### DISCUSSION TITLE
Add labels to stats

### DISCUSSION THREAD

**Discussion by CyrusV5** [devmount/third-stats#323] [Ideas]
Hello,
First of all, thank you for creating this tool!
Adding label data to the stats would be useful for people that frequently use labels to sort their emails.

**Comment by devmount** [comment]
You're welcome. Great idea! What metrics / charts are you interested in specifically? A bar chart showing the number of emails per label? A line chart showing the distribution per label over time?

**Reply by CyrusV5** [reply]
A chart representing the number of mails per label would be great!
However, adding a visual distribution over time would be awesome!

**Reply by devmount** [reply]
First thing first, and then we'll see about the latter. Since I'm not really using labels that much in Thunderbird, would you be interested and available for testing this feature before release?

**Reply by CyrusV5** [reply]
Agreed.
Yes, have some data that could be used for testing.

**Reply by devmount** [reply]
Great! I'll let you know, when a release candidate is available!

**Reply by CyrusV5** [reply]
Awesome, looking forward to it!

**Reply by devmount** [reply]
@CyrusV5 A basic version of this is implemented now with #329. You can head over to the [ThirdStats CDN](https://third-stats.cdn.devmount.com/) and download the latest build to test it. Please log any issues directly at #328, i will reopen it then.

**Comment by CyrusV5** [comment]
This just occured to me, the Favorites data would be nice to be numbered/displayed as well. I could edit the title to add the info as well instead of creating a new discussion.

**Reply by devmount** [reply]
Yes, you're welcome to edit/add that. Favorites could be visualized simply as an additional "label".

**Reply by devmount** [reply]
I've reconsidered this and think, it would be too confusing, to implement this as additional label. E.g. you cannot separate anymore if this is a label called "favorite" or an actually starred email.

So how about just adding a new number section for starred emails in the top bar? This can also hold information about how many emails are labeled at all. I've thought of something like this:

![image](https://user-images.githubusercontent.com/5441654/126036917-1b6a2655-72af-4e25-9cd8-82318d33b74d.png)

**Reply by devmount** [reply]
@CyrusV5 This is implemented now with #338. You can head over to the [ThirdStats CDN](https://third-stats.cdn.devmount.com/) and download the latest build to test it.
